app/renderer/components/download/downloadsBar.js

```diff
@@ -7,24 +7,39 @@ const React = require('react')
 // Components
 const ImmutableComponent = require('../immutableComponent')
 const Button = require('../common/button')
+const BrowserButton = require('../common/browserButton')
 const DownloadItem = require('./downloadItem')
 
+const {StyleSheet, css} = require('aphrodite/no-important')
+
 // Actions
 const windowActions = require('../../../../js/actions/windowActions')
 const webviewActions = require('../../../../js/actions/webviewActions')
+const appActions = require('../../../../js/actions/appActions')
 
 // Utils
 const contextMenus = require('../../../../js/contextMenus')
 
+const cx = require('../../../../js/lib/classSet')
+
 class DownloadsBar extends ImmutableComponent {
   constructor () {
     super()
     this.onHideDownloadsToolbar = this.onHideDownloadsToolbar.bind(this)
+    this.onShowDownloads = this.onShowDownloads.bind(this)
   }
   onHideDownloadsToolbar () {
     windowActions.setDownloadsToolbarVisible(false)
     webviewActions.setWebviewFocused()
   }
+  onShowDownloads () {
+    appActions.createTabRequested({
+      activateIfOpen: true,
+      url: 'about:downloads'
+    })
+    windowActions.setDownloadsToolbarVisible(false)
+    webviewActions.setWebviewFocused()
```

Comment:
only `webviewActions.setWebviewFocused()`. For parity, we should auto-hide the downloads bar after clicking "show all", as you did above.